src/service/save-receipt.js

```diff
@@ -0,0 +1,29 @@
+import { postRequest } from '../utils/api-utils';
+
+// eslint-disable-next-line import/prefer-default-export
+export const saveReceipt = async (data) => {
+  console.log('data', data)
```

Comment:
What is `data` used for?

Reply:
data here will be used to construct the payload, which will be handled as pat of save stock-receipt functionality card